wdl2cwl/tests/cwl_files/seqtk.cwl

```diff
@@ -0,0 +1,42 @@
+class: CommandLineTool
+id: Sample
+inputs:
+  - id: sequenceFile
+    type: File
+  - id: fractionOrNumber
+    type: float
+  - id: outFilePath
+    default: subsampledReads.fq.gz
+    type: string
+  - id: twoPassMode
+    default: false
+    type: boolean
+  - id: zip
+    default: true
+    type: boolean
+outputs:
+  - id: quality_yield_metrics
+    type: File
+    outputBinding:
+        glob: '"abc"'
+requirements:
+  - class: InitialWorkDirRequirement
+    listing:
+      - entryname: example.sh
+        entry: |4
+
+            set -e -o pipefail
+            mkdir -p "$(dirname $(inputs.outFilePath))"
+            $(inputs.preCommand)
+            seqtk sample \
+            $(inputs."-s "+seed) \
+            $(inputs.true="-2 "false=""twoPassMode) \
+            $(inputs.sequenceFile.path) \
+            $(inputs.fractionOrNumber) \
+            $(inputs.true="| gzip"false=""zip) \
```

Comment:
These aren't valid CWL expressions. As usual, you can either
1) simplfy your test to not include these advanced cases & make an issue to come back to this later
2) Fix the advanced cases now in this PR